style.css

```diff
@@ -191,6 +191,83 @@ a {
   width: 18px;
 }
 
+.card-container {
```

Comment:
- [ ] I think your desktop display of the popup has problems with positioning of elements. I present the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546&t=vcUfB8PtX4JkRFcP-0) design and your design below for clarity.
- Your design
![image](https://user-images.githubusercontent.com/62849872/210931027-d13c2125-f03d-48ec-83a9-c9aa23b3a96c.png)
- Figma design
![image](https://user-images.githubusercontent.com/62849872/210931111-0b79d27e-5a50-457a-a991-b277e7668b8f.png)
Kindly make your design and positioning of elements similar to the figma design